sdk/test-utils/perf/src/options.ts

```diff
@@ -53,6 +53,15 @@ export type PerfOptionDictionary<TOptions = Record<string, unknown>> = {
   [longName in keyof TOptions]: OptionDetails<TOptions[longName]>;
 };
 
+/**
+ * This is exactly same as PerfOptionDictionary, but the "value" is required.
```

Comment:
```suggestion
 * This is exactly same as {@link PerfOptionDictionary}, but the `value` is required.
```

These little thing make comments more enjoyable, at least. They’re also useful for doc parsers, if that ever becomes useful.